x/dualstaking/types/codec.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgDelegate{}, "dualstaking/Delegate", nil)
-	cdc.RegisterConcrete(&MsgDelegate{}, "dualstaking/Redelegate", nil)
+	cdc.RegisterConcrete(&MsgRedelegate{}, "dualstaking/Redelegate", nil)
```

Comment:
also here, I thought this was fixed elsewhere already.

Reply:
should rebase

Reply:
done